src/workerd/jsg/jsg.h

```diff
@@ -1787,6 +1787,12 @@ template <typename TypeWrapper>
 class Isolate;
 // Defined in setup.h -- most code doesn't need to use these directly.
 
+class AsyncContextFrame;
+
+#define JSG_PRIVATE_SYMBOLS(V)       \
+  V(ASYNC_RESOURCE, "asyncResource")
```

Comment:
Nitpick: Maybe should be called `asyncContext` now, since it points to an `AsyncContextFrame`, not a resource.